docs/docusaurus/docs/reference/learn/data_quality_use_cases/unstructured_data.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_label: 'Unstructured data'
+title: 'Validate unstructured data with GX Cloud'
+description: 'Learn how to validate metadata from scanned PDFs using GX Cloud'
+---
+
+Enterprise data often consists of large amounts of unstructured data such as PDFs, images, emails, and sensor logs, but many often find it difficult to validate the quality of it. Data quality issues related to unstructured data can often go unnoticed, leading to downstream problems. For example, an AI model may be compromised if duplicate documents and failed OCR (Optical Character Recognition) do not get immediately flagged, leading to poor outputs.
+
+This tutorial provides a working, hands-on example of how to validate unstructured data using sample PDF data and GX Cloud. An OCR process on a PDF doesn't just extract text; it produces metadata like confidence scores, and word counts. GX Cloud allows you to set up data quality checks on this metadata to maximize the confidence in your unstructured data, all while allowing your collaborators to view the results. 
+
+## Prerequisite Knowledge
+
+This article assumes basic familiarity with GX components and workflows. If you're new to GX, start with the [GX Cloud](/cloud/overview/gx_cloud_overview.md) and [GX Core](/core/introduction/gx_overview.md) overviews to familiarize yourself with key concepts and setup procedures.
+
+## Prerequisites
+
+- [Python version 3.9 to 3.12](https://www.python.org/downloads/)
+- A [GX Cloud account](https://greatexpectations.io/cloud)
+Your [Cloud credentials](/cloud/connect/connect_python.md#get-your-credentials) saved in your [environment variables](/cloud/connect/connect_python.md#set-your-credentials-as-environment-variables).
+
+## Install dependencies
+
+Open a terminal window and navigate to the folder you want to use for this tutorial.
+
+Install [poppler](https://poppler.freedesktop.org/) and [tesseract](https://github.com/tesseract-ocr/tesseract). Poppler is a PDF rendering library that this tutorial uses to read the PDFs. Tesseract is an open source OCR engine that this tutorial uses to from OCR on the PDFs.
+
+```bash title="Terminal input"
+brew install poppler
+brew install tesseract
+```
```

Comment:
I can't add a suggestion for this because github doesn't handle nested code fences, but code blocks should be indented under the related numbered step